scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -531,10 +531,12 @@ s32 BPF_STRUCT_OPS(rusty_select_cpu, struct task_struct *p, s32 prev_cpu,
 	 * local dsq of the waker.
 	 */
 	if (wake_flags & SCX_WAKE_SYNC) {
-		struct task_struct *current = (void *)bpf_get_current_task();
+		struct task_struct *current = (void *)bpf_get_current_task_btf();
```

Comment:
JC, why the change to this function call?

Reply:
It returns a trusted pointer, so we don't have to bother with a probed read. Probably should've put it in a separate PR.